packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -177,6 +191,45 @@ const styles = (theme) => ({
       display: "block"
     }
   },
+  nextUnreadIn: {
+    color: theme.palette.grey[800],
+    
+    [theme.breakpoints.up('md')]: {
+      position: "absolute",
```

Comment:
This sure feels very fragile. I expect we will want to refactor this somehow at some points.

Reply:
To the extent this is fragile, I think the fragility is in the conceptual layout, not anything the code is adding.